gateway/sds_gateway/api_methods/utils/metadata_schemas.py

```diff
@@ -406,14 +424,28 @@ def get_mapping_by_capture_type(
     }
 
 
-def infer_index_name(capture_type: CaptureType) -> str:
+def infer_index_name(capture_type: Any) -> str:
```

Comment:
this fn is fine to accept `str`, but we should annotate it as `capture_type: CaptureType`